### PR TITLE
[stable/mongodb-replicaset] Fix using existing Metrics Secret

### PR DESCRIPTION
**Pull request by grzegdl**
Signed-off-by: Grzegorz Dlugolecki <grzegdl@gmail.com>

#### What this PR does / why we need it:
 
Check if there is existing Secret defined for Metrics has a typo which prevents it's usage.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] title of the PR contains starts with chart name e.g. `[stable/chart]`

### CODE REVIEW

File: stable/mongodb-replicaset/templates/mongodb-metrics-secret.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{{- if and (.Values.auth.enabled) (not .Values.auth.exisitingMetricsSecret) (.Values.metrics.enabled) -}}
+{{- if and (.Values.auth.enabled) (not .Values.auth.existingMetricsSecret) (.Values.metrics.enabled) -}}
```
Comment (steven-sheehy) [inline stable/mongodb-replicaset/templates/mongodb-metrics-secret.yaml:1]: Can you fix the spelling in the values.yaml and add it to the README?

**Review by steven-sheehy — CHANGES_REQUESTED** [review]

File: stable/mongodb-replicaset/templates/mongodb-metrics-secret.yaml (hunk 1 above)
Comment (grzegdl) [reply stable/mongodb-replicaset/templates/mongodb-metrics-secret.yaml:1]: Sure, on the way!

**Comment by steven-sheehy** [comment]
/lgtm
Thanks!

**Pull request merged**